nuxeo-features/nuxeo-platform-csv-export/src/main/java/org/nuxeo/ecm/platform/csv/export/io/DocumentModelCSVHelper.java

```diff
@@ -142,6 +142,23 @@ public static Directory getVocabulary(Type type) {
         return null;
     }
 
+    public static List<String> getList(RenderingContext ctx, String key) {
+        Object value = ctx.getParameter(key);
```

Comment:
Can't we ensure that we always set a List in the context and just do a cast when we get it ?

Reply:
We could, I remove the encapsulation from String but kept null values filtering that would force checks in other code